Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -149,9 +146,9 @@ String CSSCounterStyle::counterForSystemAdditive(unsigned value) const
     }
 
     StringBuilder result;
-    auto appendToResult = [&](String symb, unsigned frequency) {
+    auto appendToResult = [&](String symbol, unsigned frequency) {
```

Comment:
I think this should take `const String&` to save a bit of reference count churn. Obviously not important to do that in this patch.

Reply:
I've addressed this in https://github.com/WebKit/WebKit/pull/13322